README.md

```diff
@@ -28,6 +28,10 @@ travis-ci-cloudfront-invalidation requires six flags:
     b: 'TravisBranch'
     p: 'TravisPullRequest'
 
+These flags are optional:
+
+    o: 'OnBranches' defaults to 'master'
+
 ## Example
 
-    travis-ci-cloudfront-invalidation -a $AWS_ACCESS_KEY -s $AWS_SECRET_KEY -c $AWS_CLOUDFRONT_DIST_ID -i '/*' -b $TRAVIS_BRANCH -p $TRAVIS_PULL_REQUEST
+    travis-ci-cloudfront-invalidation -a QDUAIUQ3OQ7I2WDNLUCO, -s f4GTRs34nRS3Ntf678lDsT87Qszvrs23lmREd57 -c Y54SDKE34MDTU1 -i '/*' -b $TRAVIS_BRANCH -p $TRAVIS_PULL_REQUEST -o 'master,stage'
```

Comment:
Did you just commit your own AWS credentials ? 🤔

Reply:
you scared me there for a second. 

But I think I randomized them a bit. it's just to help people identify what the key and secret looks like.

Reply:
Ouch, glad these weren't your actual credentials!